config.py

```diff
@@ -222,6 +222,8 @@ def _checksd_path(directory):
 
 
 def _is_affirmative(s):
+    if s is None:
```

Comment:
I agree but for some reason I find things more readable the way they are, so I'll ignore this